.github/workflows/python-tests.yml

```diff
@@ -23,6 +23,7 @@ jobs:
     strategy:
       matrix:
         python-version: ["3.6", "3.9"]
+        test_repository: [" -k 'not RepositoryTest'", "test_repository.py -k RepositoryTest"]
```

Comment:
That's practical, but it results in 
<img width="313" alt="image" src="https://user-images.githubusercontent.com/30755778/153671173-f6a07508-1478-4c2b-a96a-6d146f50d4bb.png">

which is a bit unreadable. Do you think this could be clarified?